pom.xml

```diff
@@ -169,6 +169,11 @@
         <version>${guava.version}</version>
         <scope>runtime</scope>
       </dependency>
+      <dependency>
+        <groupId>org.apache.httpcomponents</groupId>
```

Comment:
is this one needed instead of other http clients? Or is this just overriding the version of the spring boot dependency?

Reply:
It's just overriding the version of the spring boot dependency until a new spring boot starter parent will be released and will contain the latest dependencies.